.github/workflows/publish_docs.yml

```diff
@@ -15,9 +15,9 @@
 name: Publish Documentation
 
 on:
-  push:
-    branches:
-      - main
+  schedule:
+    # Every working day at 12pm UTC
+    - cron: '0 12 * * 1-5'
```

Comment:
Will this still pull from `main` (the default branch) if the branch is not specified by the `push` trigger?

Reply:
I think so? But would be nice to get confirmation from @GMNGeoffrey.

Reply:
Yes

```
# The branch, tag or SHA to checkout. When checking out the repository that
# triggered a workflow, this defaults to the reference or SHA for that event.
# Otherwise, uses the default branch.
ref: ''
```
https://github.com/actions/checkout#usage